Packs/CloudShare/Integrations/CloudShare/CloudShare.yml

```diff
@@ -0,0 +1,2683 @@
+commonfields:
+  id: CloudShare
+  version: -1
+vcShouldKeepItemLegacyProdMachine: false
+name: CloudShare
+display: CloudShare
+category: Utilities
+description: Cloudshare integration.
+configuration:
+- display: Hostname
+  name: hostname
+  defaultvalue: use.cloudshare.com
+  type: 0
+  required: true
+  additionalinfo: This is usually use.cloudshare.com unless your using some mock or
+    proxy
+- display: API ID
+  name: api_id
+  type: 4
+  required: true
+  additionalinfo: Valid CloudShare API ID
+- display: API Key
+  name: api_key
+  type: 4
+  required: true
+  additionalinfo: Valid CloudShare API ID
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+script:
+  script: ''
+  type: python
+  commands:
+  - name: cloudshare-get-envs
+    arguments:
+    - name: owned
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Returns only environments owned by the user.
+      defaultValue: "false"
+    - name: visible
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Returns all environments visible to the user.
+      defaultValue: "false"
+    - name: ownerEmail
```

Comment:
Command arguments should follow conventions, (i.e. alert_id= instead of AlertID=).

Please go over all the arguments and change them to snake case (`_` between words).